scripts/export_json.py

```diff
@@ -0,0 +1,247 @@
+
```

Comment:
Copyright?

Reply:
I have made comments on this file, but I think you've actually deprecated this code, so we may as well remove this code if it's not used anymore.